docs/integration-branch-cli-setting.md

```
@@ -0,0 +1,173 @@
+# `integrationBranch` CLI setting (AIWO-039)
+
+## Defect
+
+`fn settings set integrationBranch <branch>` (and `fn settings set integrationBranch <branch> --project <name>`)
+previously failed with:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)